web/pgadmin/static/js/components/ReactCodeMirror/components/Editor.jsx

```diff
@@ -1,407 +1,414 @@
-/////////////////////////////////////////////////////////////
```

Comment:
Again, why is this file showing as everything changed? Impossible for me to review this.